### PR TITLE
Tmedia 724 header nav responsive feedback

### PR DESCRIPTION
**Pull request by JackHowa**
- this is a feedback pr for the #1386 implementing some of the feedback
- Fixes stylelint issue
- Updates for the upcoming merge
- Uses classnames with modifiers rather than setting the height in-line. Also fixes a code quality alert
- Remove unused token declarations

### CODE REVIEW

**Comment by JackHowa** [comment]
merging in upstream data is too messy; will cherry-pick updates

File: blocks/header-nav-chain-block/_index.scss [hunk 1]
```diff
@@ -0,0 +1,250 @@
+@use "@wpmedia/arc-themes-components/scss";
+
+.b-header-nav-chain {
+	&--scrolled {
+		@include scss.block-components("header-nav-chain-nav-chain-scrolled");
+		@include scss.block-properties("header-nav-chain-nav-chain-scrolled");
+	}
+	&--not-scrolled {
```
[the remark this answers is not in the document]
Comment (JackHowa) [reply blocks/header-nav-chain-block/_index.scss:9]: updated

**Review by brentmiller1973 — CHANGES_REQUESTED** [review]
@JackHowa Appreciate the work, but a simple isScrolled var does not correctly implement the behavior of minimizing the height of the nav from its starting height once the user starts scrolling.

**Comment by JackHowa** [comment]
> Appreciate the work, but a simple isScrolled var does not correctly implement the behavior of minimizing the height of the nav from its starting height once the user starts scrolling.

I'm not sure what you mean. It will apply the style of whatever is set to "header-nav-chain-scrolled". There's an event listener for on scroll behavior. Based on previous discussions and work, we want to have the customer set the height and, by extension, offset in the theme file, right? Setting in-line styles is currently overriding what's in the theme file that @matthewroach added. 

@visionbegin

**Comment by JackHowa** [comment]
used as reference in #1386

**Pull request closed without merging**